### PR TITLE
create customizable zoomRange for loading imagery with a small max zoom level

### LINKED ISSUE #1481: Allow imagery minZoom variable to be overriden in imagery json data
### Description

Right now minZoom is hardcoded to be currentZoom - 5 in https://github.com/facebook/Rapid/blob/main/modules/pixi/PixiLayerBackgroundTiles.js#L164. For overlays and imagery that don't provide data up to zoom level 18, this causes them to disappear at higher zoom levels. It would be nice for this to be configurable so that they don't disappear.

### PR DESCRIPTION
**Pull request by emersonveenstra**
fixes #1481 by removing the hardcoded 5 zoom limit with a configurable one with a default of 5

### CODE REVIEW

**Comment by Bonkles** [comment]
No arguments here! @bhousel thoughts?

File: scripts/update_imagery.js [hunk 1]
```diff
@@ -158,6 +158,10 @@ for (const [sourceID, source] of sources) {
     ];
   }
 
+  if (source.zoomRange) {
```
Comment (bhousel) [inline scripts/update_imagery.js:161]: Looking through the [editor-layer-index schema](https://github.com/osmlab/editor-layer-index/blob/gh-pages/schema.json) I don't think we would get source imagery with a `zoomRange` property, so this block would never run?

**Comment by bhousel** [comment]
> No arguments here! @bhousel thoughts?

Seems ok to me, thanks @emersonveenstra   👍 
(Let me know if whether you really need the lines in `update_imagery.js`)

**Comment by emersonveenstra** [comment]
I added those lines so that manual imagery could use it as well, if that's not something you want i can remove it

File: scripts/update_imagery.js (hunk 1 above)
Comment (emersonveenstra) [reply scripts/update_imagery.js:161]: oops, didn't see this, see https://github.com/facebook/Rapid/pull/1496#issuecomment-2263168116 for the reasoning

**Comment by bhousel** [comment]
> I added those lines so that manual imagery could use it as well, if that's not something you want i can remove it

Its ok, we can keep it in!  (Was more worried that ELI changed something without telling us.)

**Pull request merged**